core/tests/puppeteer-acceptance-tests/utilities/user/question-submitter.ts

```diff
@@ -491,15 +500,42 @@ export class QuestionSubmitter extends BaseUser {
     showMessage('Image interaction has been added successfully.');
   }
 
+  // TODO (#22539): This function has a duplicate in exploration-editor.ts.
+  // To avoid unexpected behavior, ensure that any modifications here are also
+  // made in editDefaultResponseFeedback() in exploration-editor.ts.
   /**
-   * Function to edit the default response feedback.
+   * Function to add feedback for default responses of a state interaction.
+   * @param {string} defaultResponseFeedback - The feedback for the default responses.
+   * @param {string} [directToCard] - The card to direct to (optional).
+   * @param {string} [directToCardWhenStuck] - The card to direct to when the learner is stuck (optional).
    */
-  async editDefaultResponseFeedback(): Promise<void> {
-    await this.clickOn(defaultResponseTab);
-    await this.clickOn(openOutcomeFeedBackEditorSelector);
-    await this.page.waitForSelector(stateContentInputField, {visible: true});
-    await this.type(stateContentInputField, 'Wrong Answer');
-    await this.clickOn(saveOutcomeFeedbackButton);
+  async editDefaultResponseFeedbackInQuestionSubmitter(
```

Comment:
Ditto. Also I think you'll want to call the other function ...InExplorationEditorPage, similarly.

Reply:
Done.